packages/itwinui-css/src/bridge.scss

```diff
@@ -245,35 +282,117 @@
     }
   }
 
+  /* ---------------------------------------------------------------------------- */
+  /* Icon */
+
+  .iui-svg-icon {
+    // StrataKit icons are text color by default
+    --_iui-svg-icon-default-fill: currentColor;
+  }
+
   /* ---------------------------------------------------------------------------- */
   /* Input */
 
   .iui-input:not([role='combobox']) {
     // StrataKit input background colors are different from global values
-    --_iui-field-color-background: var(--ids-color-bg-control-textbox);
+    --_iui-field-color-background: var(--ids-color-bg-page-base);
```

Comment:
This might be a mistake/missing change in StrataKit. Let's leave it as `color-bg-control-textbox`.